### PR TITLE
Update required node version to V12 & npm install command

### PR DESCRIPTION
**Pull request by Darqam**
**Please describe the changes this PR makes and why it should be merged:**
Fairly self explanatory, guide should not state improper nodejs requirement. Minimum required nodejs version is changed from 10.X in the guide about updated to djs V12 to 12.x

edit: Also changing install instruction since V12 is now standard release

### CODE REVIEW

File: guide/additional-info/changes-in-v12.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ After a long time in development, Discord.js v12 is nearing a stable release, me
 
 ## Before You Start
 
-v12 requires Node 10.2.x or higher to  use, so make sure you're up-to-date.  To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it!  There are many resources online to help you get up-to-date.
+v12 requires Node 12.x or higher to  use, so make sure you're up-to-date.  To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it!  There are many resources online to help you get up-to-date.
 
 For now, you do need Git installed and added to your PATH environment, so ensure that's done as well - again, guides are available online for a wide variety of operating systems.  Once you have Node up-to-date and Git installed, you can install v12 by running `npm install discordjs/discord.js` in your terminal or command prompt for text-only use, or `npm install discordjs/discord.js @discordjs/opus` for voice support.
```
Comment (Jiralite) [inline guide/additional-info/changes-in-v12.md]: How about stating this way to install: `npm install discord.js`?

Comment (Darqam) [reply guide/additional-info/changes-in-v12.md]: Yeah that's not a bad idea, seems silly to make another PR just for that. Changed and will update description+title

File: guide/additional-info/changes-in-v12.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ After a long time in development, Discord.js v12 is nearing a stable release, me
...
+v12 requires Node 12.x or higher to  use, so make sure you're up-to-date.  To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it!  There are many resources online to help you get up-to-date.
 
-For now, you do need Git installed and added to your PATH environment, so ensure that's done as well - again, guides are available online for a wide variety of operating systems.  Once you have Node up-to-date and Git installed, you can install v12 by running `npm install discordjs/discord.js` in your terminal or command prompt for text-only use, or `npm install discordjs/discord.js @discordjs/opus` for voice support.
+For now, you do need Git installed and added to your PATH environment, so ensure that's done as well - again, guides are available online for a wide variety of operating systems.  Once you have Node up-to-date and Git installed, you can install v12 by running `npm install discordjs/discord.js` in your terminal or command prompt for text-only use, or `npm install discord.js @discordjs/opus` for voice support.
```
Comment (Anish-Shobith) [inline guide/additional-info/changes-in-v12.md]: you can install v12 by running `npm install discord.js`, update there too , its only updated in the case of the voice state

Comment (Anish-Shobith) [reply guide/additional-info/changes-in-v12.md]: And I don't think you need git to install v12 now ? as its stable ?

Comment (Darqam) [reply guide/additional-info/changes-in-v12.md]: ye, I'm blind (or realistically didn't bother to read/use ctrl+f) thanks

**Comment by Danktuary** [comment]
Would you mind going around and updating the other files that currently state 10.2.x as the version number too? There's a couple instances in the `/preparations/README.md` file and one in the `/commando/README.md` file

**Pull request merged**